discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -75,6 +75,14 @@ def _terraform_params(self, **kwargs):
                 params[key] = value
         return Munch.fromDict(params)
 
+    def _resource_param(self, param_name, default=None, **kwargs):
```

Comment:
Please reuse the other implementation of `_resource_param`.

See [my previous comment](https://github.com/openshift/assisted-test-infra/pull/707#discussion_r614743731)

Reply:
We can't reuse since we obtain operators differently and values are in different format.

Reply:
The function doesn't have to obtain the operators on its own, it can receive them as a parameter (I showed how in my previous comment). Then each caller can provide those parameters differently. 

Whatever conversions need to be done for the formats to match should be done in a wrapper function - the summing logic should not be repeated

Reply:
I disagree with you. We are trading small reuse for clarity but you are the maintainer so I updated.

Reply:
It's fair to disagree, didn't mean to force you to make the change - just wanted to make sure the change I was looking for is understood and considered.

And I agree that reuse has a clarity cost, I personally tend to be on the reuse side. Thank you for making the change regardless.